project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -29,6 +29,8 @@
 
 *Please provide a link to a document which describes the release process. This should explain how changes are communicated, versioning schemes, signing of release artifacts, etc.*
 
+*You can find an example here: https://osrd.fr/en/docs/guides/release/process/*
```

Comment:
I would suggest to link to https://osrd.fr/en/docs/guides/release/. The branch structure is important, but the steps how to cut a release, create changelogs, as well, I would also see as part of the release process.